tests/e2e/unified_test.go

```diff
@@ -794,3 +794,98 @@ func buildCRDFilter(keepCRDs map[schema.GroupKind]bool) create.HarnessOption {
 		return keepCRDs[gk]
 	})
 }
+
+func TestAllInSeries_IAM(t *testing.T) {
```

Comment:
Yes correct we COULD run everything from TestAllInSeries and we probably should and will in the end but I wanted to start with a separate entry point for now so I can tweak it as needed for IAM. In the end we can collapse it to one if it makes sense.